modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,289 @@
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.bytes
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.size
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.async_put
+
+|<prefix>.causal_clustering.core.raft_log_entry_prefetch_buffer.sync_put
+
+|<prefix>.causal_clustering.core.message_processing_delay
+
+|<prefix>.causal_clustering.core.message_processing_timer
+
+|<prefix>.causal_clustering.core.replication_new
+
+|<prefix>.causal_clustering.core.replication_attempt
+
+|<prefix>.causal_clustering.core.replication_fail
+
+|<prefix>.causal_clustering.core.replication_maybe
+
+|<prefix>.causal_clustering.core.replication_success
+
+|<prefix>.causal_clustering.core.last_leader_message
+
+a|**link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#read-replica-metrics[Read Replica metrics] - replaced accordingly by the link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#store-copy-metrics[Store copy metrics]**
+
+|<prefix>.causal_clustering.read_replica.pull_updates
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_requested
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_received
+
+|===
+====
+
+=== SSL framework
+
+.See the CBC-based ciphers that are no longer used from Neo4j 2025.01:
+
+* TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+* TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_128_CBC_SHA256
+
+For more details, refer to the page on link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-other-configs[SSL framework] under the Security section.
+
+.Private keys
+
+Versions before Neo4j 5.0 allow keys to be stored with the old PKCS #1 standard.
+You can identify them by the line `-----BEGIN RSA PRIVATE KEY-----` at the beginning of the file.
+While Neo4j 5.0 can load and use those keys, they are considered deprecated and will be removed in a future version.
+
+
+.SSL-related configuration setting (or move it to the section 'Changed default values'?)
+
+Starting from 2025.01, the default value of `dbms.ssl.policy.*.verify_hostname` is changed from `false` to `true`.
+See details in the table link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-configuration[in the table under Configuration section on the page SSL framework].
+
+=== Ports
+
+Port `5000` is no longer used from 2025.01 onwards.
+
+=== Deprecated built-in procedures
+
+.See all deprecated procedures
+[%collapsible]
+====
+[options=header, cols="3m,1,1,3"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+| Comment
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_quarantineDatabase[`dbms.quarantineDatabase()`] label:admin-only[]
+|
+| {check-mark}
+| Replaced by xref:procedures.adoc#procedure_dbms_unquarantineDatabase[`dbms.unquarantineDatabase()`]
+|===
+====
+
+=== Java API
```

Comment:
I don't know where to take information. I believe someone from Kernel may help

Reply:
Misha can help with the Java API.